CampaignAutomation/Tests/CampaignAutomationImplementingTests.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  CampaignAutomationImplementingTests.swift
+//  RealifeTechTests
+//
+//  Created by Ross Patman Work on 08/12/2021.
+//  Copyright © 2021 Realife Tech. All rights reserved.
+//
+
+import XCTest
+@testable import RealifeTech
+
+class CampaignAutomationImplementingTests: XCTestCase {
+
+    private var mockAnalytics: MockAnalytics!
+    private var sut: CampaignAutomationImplementing!
+
+    override func setUpWithError() throws {
```

Comment:
Could we call `try super.setUpWithError()` here please? Thank you

Reason being that Apple doesn't guarantee there won't be changes in the superclass, so it's best to keep calling super. :) (Discussion: https://developer.apple.com/forums/thread/113314)

Reply:
Done.